gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/converters/StringToAttributeKindConverter.java

```diff
@@ -29,14 +29,30 @@ public Value doConvert(String value, AttributeKind attributeKind, Value.Builder
 
       case TYPE_DOUBLE:
         valueBuilder.setValueType(ValueType.DOUBLE);
-        valueBuilder.setDouble(Double.parseDouble(value));
+        /*
```

Comment:
Just curious. What happens in the TYPE_BOOL case?

Reply:
It would be false by default if the value is NOT "true".